src/posix/platform/include/openthread/openthread-system.h

```diff
@@ -176,6 +176,14 @@ const char *otSysGetThreadNetifName(void);
  */
 unsigned int otSysGetThreadNetifIndex(void);
 
+/**
+ * This method returns the infrastructure network interface name.
+ *
+ * @returns The infrastructure network interface name.
+ *
+ */
+const char *otSysGetInfraNetifName(void);
```

Comment:
The `unsigned int otSysGetThreadNetifIndex(void);` API returns the index of the Thread TUN interface. To make the APIs symmetric, should we unify both return values?

Reply:
I think it depends on the requirement. I only need to get the name of the infra network interface now. If we need to get the index of the infra network interface in the future, we can add it later. Thoughts?